packages/devtools_app/lib/src/timeline/flutter/timeline_processor.dart

```diff
@@ -825,17 +318,22 @@ class FullTimelineProcessor extends TimelineProcessor {
   }
 
   void _handleDurationBeginEvent(TraceEventWrapper eventWrapper) {
-    final current = _currentDurationEventNodes[eventWrapper.event.threadId];
+    final current = currentDurationEventNodes[eventWrapper.event.threadId];
     final timelineEvent = SyncTimelineEvent(eventWrapper);
+    if (current == null) {
+      if (eventWrapper.event.name.contains(uiEventName)) {
+        // pending frame . ui event flow = timeline event
+      } else if (eventWrapper.event.name.contains(gpuEventName)) {}
```

Comment:
remove no-op else case and change if to `if (current == null && eventWrapper.event.name.contains(uiEventName))`

Reply:
this slipped into the commit by accident. removed the if else clause